Index: scripts/customize.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/customize.js b/scripts/customize.js
--- a/scripts/customize.js	(revision 95e1869aec94def32d9a2d3f34c7adc34a511289)
+++ b/scripts/customize.js	(date 1620349937188)
@@ -1,0 +1,34 @@
+/*jshint esversion: 6 */
+let hair = 1;
+let skin = 0;
+let suit = 0;
+let num = 50;
+let image = document.getElementById('character');
+
+function hairPrev() {
+    "use strict";
+    if (10 === num) {
+        num = 100;
+        image.setAttribute("src", "https://via.placeholder.com/" + num);
+    }
+    else {
+        num -= 10;
+        image.setAttribute("src", "https://via.placeholder.com/" + num);
+    }
+}
+
+function hairNext() {
+    "use strict";
+    if (100 === num) {
+        num = 10;
+        image.setAttribute("src", "https://via.placeholder.com/" + num);
+    }
+    else {
+        num += 10;
+        image.setAttribute("src", "https://via.placeholder.com/" + num);
+    }
+}
+
+<button id="hairPrev" onClick="hairPrev()">Prev</button>
+<img id="character" src="https://via.placeholder.com/50" alt="character image">
+    <button id="hairNext" onClick="hairNext()">Next</button>
\ No newline at end of file
Index: customize.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\">\r\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\r\n    <title>Save Raincity - Customize</title>\r\n</head>\r\n<body>\r\n    <script src=\"scripts/customize.js\"></script>\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/customize.html b/customize.html
--- a/customize.html	(revision 95e1869aec94def32d9a2d3f34c7adc34a511289)
+++ b/customize.html	(date 1620349937194)
@@ -7,6 +7,7 @@
     <title>Save Raincity - Customize</title>
 </head>
 <body>
+
     <script src="scripts/customize.js"></script>
 </body>
 </html>
\ No newline at end of file
